Consider QR Links for paper-based presentation (#49)
smart-on-fhir/health-cards: docs/index.md

    ```
---

### Paper-based fallback: Health Card Links
* Allows basic use of health cards for users without a smartphone

> * Allow smartphone-enabled users to print a usable backup

Reviewer: ```suggestion
* Allow smartphone-enabled users to print a usable paper backup
```

* Allows full health card contents to be shared with a verifier

> * Requires that the verifier is online

Reviewer: In other paper-based credential discussions I've heard approaches where the verifier could be offline with a cached list of trusted issuers and their public keys. The verifier could periodically connect to the network to refresh its cached list, but it wouldn't necessarily have to be online at the time of verification. There are trade-offs here when it comes to having an update to date list of trusted issuers.

I don't have a strong opinion on changing this bullet because it might be more complicated/confusing than it is worth.

* Does not capture a digital record of a signed request (no DID-SIOP Request flow)
* Does not capture a digital record of the presentation  (no Verifiable Presentation flow)

> In this flow, the user obtains a physical or virtual *Health Card Link* QR code. This code can be shared with verifying parties or loaded into a new wallet app by scanning the QR code. The *Health Card Link* is a QR code whose payload consists of a minified JSON payload with three properties:

Reviewer: ```suggestion
In this flow, the user obtains a physical or virtual *Health Card Link* QR code. This code can be shared with verifying parties or loaded into a wallet app by scanning the QR code. The *Health Card Link* is a QR code whose payload consists of a minified JSON payload with three properties:
```

1. `type` value of `https://smarthealth.cards#1`, identifying this QR code as a *Health Card Link*
1. `url` value that dereferences to an encrypted Health Card (following the same JWE-based encryption scheme described above)
1. `jwk` value conveying a private key (`"alg": "ECDH-ES", "crv": "P-256"`) capable of decrypting the encrypted Health Card

Consumers can obtain a Health Card Link through two main workflows:

> * **For consumers *without* a smartphone**, issuers or other trusted "Health Card Link Creators" can create a QR code and print it. In this process, the Health Card Creator generates or obtains a Health Card; generates an encryption key; encrypts the Health Card; and causes the encrypted Health Card to be hosted at some URL. The Health Card Creator then populates a Health Card Link with the three required proeprties, printing the resulting QR code or otherwise sharing it with the consumer.

Reviewer: ```suggestion
* **For consumers *without* a smartphone**, issuers or other trusted "Health Card Link Creators" can create a QR code and print it. In this process, the Health Card Creator generates or obtains a Health Card; generates an encryption key; encrypts the Health Card; and hosts the encrypted Health Card at some URL. The Health Card Creator then populates a Health Card Link with the three required properties, printing the resulting QR code or otherwise sharing it with the consumer.
```

Would it be less confusing to use the term _Issuer_ rather than _Creator_ here? i.e. "Health Card Link Issuer"

Author: Maybe; I want to avoid using "issuer" in two contexts though.
